app/views/group/view_groups.html.erb

```diff
@@ -21,7 +21,7 @@
 				<div class="troop-functions">
 					<!--<tr><td><a href=""> Routes </a></td><td><a href=""> Invoices </a></td></tr>-->
 					<li id="invoice-link"><%= link_to view_invoices_url(:group_id => troop.id) do %> View Invoices<% end %></li>
-					<li id="routes-link"><%= link_to view_route_url(:group_id => troop.id) do %> View Routes<% end %></li>
+					<%= active_link_to 'Routes', view_routes_url, :wrap_tag => :li %>
```

Comment:
@KReg27 What is this? The active_link_to gem was meant to be used in the titlebar. This doesn't do what you expect.